sentry-core/src/main/java/io/sentry/core/Scope.java

```diff
@@ -77,7 +77,11 @@ public void setExtra(String key, String value) {
 
   @Override
   protected Scope clone() {
-    // TODO: clone me
-    return new Scope();
+    Scope d = null;
+    try {
+      d = (Scope) super.clone();
+    } catch (CloneNotSupportedException ignored) {
+    } // Won't happen
```

Comment:
Can we Log `ERROR` here? I love when I see _can't happen_ in the logs 😄

Reply:
in this case, I'd throw `CloneNotSupportedException` and let `Hub` log it then.
I just did the same as you did before when calling `timestamp.clone()` it returns null if an error has occurred.

Reply:
Agreed. We have to capture all exception on the `Hub` and log them out. We don't bubble any runtime errors to the users (our SDK **can't** crash apps). We can crash during initialization but not during execution.